Automattic/wp-calypso
# Site Settings: Move Related Posts in a separate component

As #11615 suggests, we'll move several settings around in order to create a new Traffic settings section. This requires several preparations. 

This PR suggests that we move the Related Posts card into a separate component, so we will be able to embed it easily within settings sections. This will also allow us to separate the component code and styles from the ones that belong to the main general form (which is already huge). In addition, the PR updates some CSS class names in order for them to follow our standards.

To test:
* Checkout this branch
* Go to `/settings/general/$site` and verify there are no visual or functional differences with the Related Posts card and settings:
  * Test with a Jetpack site
  * Test with a WordPress.com site
